print('---------1번---------')
a = (lambda x,y: x ** y)(1, 2)
print(a)
(lambda x,y: x**y)(3,2)
f=lambda x,y: x**y ; f(3,2)
print('---------2번---------')
ex=[1,2,3,4,5]
b = list(map(lambda x : x**2,ex))
print(b)
print('---------3번---------')
print('3')
print('---------4번---------')
print('3')
print('---------5번---------')
print('2')
print('---------6번---------')
print('import fah_converter as fah')
print('\n---------7번---------')
print('from game.graphic.render import render_test()')
print('---------8번---------')
print("(가) a")
print('(나) write')
print('---------9번---------')
print("(가) 쓰기 모드 : 'w'")
print("(나) 추가 모드 : 'a'")
print("(다) 읽기 모드 : 'r'")
print('---------10번---------')
print('from calculator import *')
print('---------11번---------')
print('732111종료되었습니다. (결과값은 한줄씩)')
print('---------12번---------')
print('5')
print('---------13번---------')
print('4')
print('---------14번---------')
print('4')
#SyntaxError: 문법오류
print('---------15번---------')
print('4')
print('---------16번---------')
print('"숫자가 아닙니다."')
print('---------17번---------')
print("1 없는 변수에 접근할 때: NameError"
      "\n2 파일 처리에서 오류가 발생할 때: IOError" #파일처리 > input/output 관련
      "\n3 실행시에 오류가 발생할 때: RuntimeError"
      "\n4 딕셔너리에 키가 없을 때: KeyError")
print('---------18번---------')
print('Not divided by 0   1,3   2,1   (한줄씩 출력)')
print('---------19번---------')
print('with open("i_have_a_dream.txt","r") as f: \n    f.read(contents) \n    print(contents)')
with open("i_have_a_dream.txt","r") as f:
      contents = f.read()
      print(contents)
